src/Presentation/Presentation.csproj

```diff
@@ -2,6 +2,11 @@
 
     <PropertyGroup>
         <TargetFramework>net5.0</TargetFramework>
+        <Version>2021.0.0</Version>
     </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.AspNetCore.Mvc.Versioning" Version="5.0.0" />
+    </ItemGroup>
```

Comment:
Are we using this new referenced package? If no, should we consider removing it from references?